integration-tests/jpa-without-entity/pom.xml

```diff
@@ -105,6 +105,7 @@
                                     <cleanupServer>true</cleanupServer>
                                     <enableHttpUrlHandler>true</enableHttpUrlHandler>
                                     <graalvmHome>${graalvmHome}</graalvmHome>
+                                    <enableJni>false</enableJni>
```

Comment:
I'm not sure I understand why you are adding all those lines?

Reply:
Well, I'm only restoring values that existed before we moved to GraalMV `19.3.0` on our `master` branch. None of these `enableJni` is new and maybe most (if not all) of them are irrelevant.